src/azure-cli-core/azure/cli/core/azlogging.py

```diff
@@ -173,7 +175,12 @@ def log_cmd_metadata_extension_info(self, extension_name, extension_version):
             self.command_metadata_logger.info("extension name: %s", extension_name)
             self.command_metadata_logger.info("extension version: %s", extension_version)
 
-    def end_cmd_metadata_logging(self, exit_code):
+    @staticmethod
+    def deinit_cmd_metadata_logging(cli_ctx):
+        self = cli_ctx.logging
+        self.end_cmd_metadata_logging(cli_ctx.result.exit_code if cli_ctx.result else 128)
```

Comment:
>self [](start = 8, length = 4)

why not just use cli_ctx.logging.end_cmd_metadata_logging

Reply:
fixed